helpers/fakerenderer/fakerenderer.go

```diff
@@ -27,7 +29,7 @@ func (fr *FakeRenderer) GetID() string {
 }
 
 // Value returns a blank string
```

Comment:
Not really true anymore since we return a resource.Value, and it's not necessarily empty.